samples/template-operator/Makefile

```diff
@@ -69,6 +69,9 @@ operator/docker-build:
 	IMG=$(IMG) $(MAKE) -C operator/ docker-build
 .PHONY: operator/docker-push
 operator/docker-push:
+ifneq (,$(GCR_DOCKER_PASSWORD))
```

Comment:
related document description https://github.com/kyma-project/lifecycle-manager/pull/133/files#r971913470

Reply:
why not implement this logic inside the operator makefile?